config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "items", to: "items#index"
+  get "customers/:id", to: "customers#show"
+
+  post "customer_items/", to: "customer_items#create"
```

Comment:
It's conventional to start these paths with a "/" so it would be get "/items" rather than just "items"

Reply:
because it is the path that's appended onto localhost:3000 or the base URL of the application